### ISSUE TITLE
Be able to read data from disk between restarts

### ISSUE THREAD

**Issue by kahuang** [foundry-rs/foundry#3665]
### Component

Anvil

### Describe the feature you would like

Anvil will persist state to disk after hitting a certain limit. It would be nice if anvil was able to read this data / persist it between restarts. That way I could run, e.g. a game on anvil, then spin it down when it was no longer being played, and spin it up again when it was.

### Additional context

I think what this would involve is:

- Ability to specify a data directory, where data is persisted (internal state, etc.)
- Reading from the data directory if path is specified (and bootstrapping state from it on startup)

**Comment by fubhy** [comment]
Agreed. This would be super valuable for using anvil in a "full stack" development environment.
